src/state/me.ts

```diff
@@ -36,7 +37,9 @@ export const fetchPractitionerData = createAsyncThunk<MeState, void, { state: Ro
       const { email, name } = jwt_decode<{ email: string; name?: string }>(idToken)
       const practitioner = await fetchPractitioner(email)
       if (practitioner) {
+        localStorage.setItem(USERNAME_HEADER, email)
         localStorage.setItem(PRACTITIONER_ID, practitioner.id)
+        await openApiBackSession()
```

Comment:
What is this for?

Reply:
Our highly secure backend auth a user at the first request and open a session by creating a session cookie and a csrf token cookie. I need to open a Django session for the next requests.

Reply:
Do we need to open a new session each time?